CMakeLists.txt

```diff
@@ -248,7 +249,8 @@ add_custom_command(TARGET augyang
 #
 # installation
 #
-
+install(TARGETS srds_augeas DESTINATION ${CMAKE_INSTALL_LIBDIR}/sysrepo/plugins)
+install(TARGETS augyang ay_startup DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
I suppose installing `srds_augeas` as part of standard installation makes sense but it currently in `scripts/install_modules.sh` and performed using `sysrepoctl`, which is the safer way (the directory can be changed). Regarding the binaries, not sure if any of them need to be installed, they are not required for the whole project to work but I am fine with that.